### DISCUSSION TITLE
Finding the downloaded fics?

### DISCUSSION THREAD

**Discussion by helpmeimdum** [nianeyna/ao3downloader#39] [Q&A] [answered]
Alright so I went and downloaded(?) some fics but now... where would they be located at?

**Comment by nianeyna** [comment, marked as answer]
If you ran the script successfully and didn't see any errors, there should be a folder called "downloads" located in the same folder as the script itself which will have your files in it. If it's not there or it's empty then something went wrong! But hopefully it's there :)

**Reply by helpmeimdum** [reply]
Awh dang, something went wrong then. Nada. -sigh- time to see what I did wrong

**Reply by helpmeimdum** [reply]
NEVERMIND I COPIED AND PASTED THE LINK AND NOW IM GETTING THEM IN MY FOLDER IM SO HAPPY, I LOVE YOU INTERNET STRANGER, YOU'RE THE BEST.

**Reply by nianeyna** [reply]
yay! I was so confused about what could be happening if you didn't see any errors lol, glad you got it to work!

(also, thank you so much for using the discussion board for this! I really appreciate it)
